frontend/simple/views/containers/Proposals.vue

```diff
@@ -0,0 +1,205 @@
+        <voting
+          v-if="showOtherProposals"
+          v-for="proposal in groupProposals.alreadyVoted"
+          :proposal="proposal"
+          :onVoteAgainst="handleVoteAgainst"
+          :onVotedFor="handleVoteFor"
+        />
+      </div>
+      <button class="modal-close is-large" aria-label="close" @click="showOtherProposals = false"></button>
+    </div>
+
+  </dashboard-section>
+</template>
+<style lang="scss" scoped>
+@import "../../assets/sass/theme/index";
+
+.gi-notify {
+  display: block;
+  margin-top: $gi-spacer-lg;
+}
+
+.gi-showAll {
+  position: absolute;
+  top: $gi-spacer;
+  right: 0;
+}
+</style>
+<script>
+import DashboardSection from '../components/DashboardSection.vue'
+import Voting from '../components/Voting.vue'
+import VotingBanner from './VotingBanner.vue'
+
+export default {
+  name: 'Proposals',
+  components: {
+    DashboardSection,
+    Voting,
+    VotingBanner
+    // ShowProposals
+  },
+  props: {
+    proposals: Object
+  },
+  data () {
+    return {
+      showOtherProposals: false,
+      mockProposals: [
+        {
+          type: 'member',
+          member: {
+            picture: 'assets/images/default-avatar.png',
+            name: 'Gil'
+          },
+          title: 'Remove Member',
+          text: 'Sam had proposed to <strong>remove Gil</strong> from the group.',
+          ctas: {
+            for: 'Remove Gil',
+            against: 'Keep Gil'
+          },
+          members: 7,
+          votes: 4,
+          userVote: null
+        },
+        {
+          type: 'rule',
+          value: 0.6,
+          title: 'Rule: Add Memmber',
+          text: 'Karl had proposed to change <strong>Add Member Rule from 80% to 60% </strong>.',
+          textDetails: 'Instead of 6, now at least 5 of 8 members need to approve a new member.',
+          ctas: {
+            for: 'Change to 60%',
+            against: 'Keep 80%'
+          },
+          members: 7,
+          votes: 0,
+          userVote: null
+        },
+        {
+          type: 'rule',
+          value: 0.92,
+          title: 'Rule: Change Mincome',
+          text: 'Karl had proposed to change <strong>Mincome Rule from 87% to 92%</strong>.',
+          textDetails: 'Instead of 6, now at least 5 of 7 members need to approve a new member.',
+          ctas: {
+            for: 'Change to 92%',
+            against: 'Keep 87%'
+          },
+          members: 7,
+          votes: 1,
+          userVote: false
+        },
+        {
+          type: 'member',
+          member: {
+            picture: 'assets/images/default-avatar.png',
+            name: 'Kim'
+          },
+          title: 'Add Member',
+          text: 'Rachel had proposed to <strong>add Kim</strong> to the group.',
+          ctas: {
+            for: 'Invite Kim',
+            against: 'Don\'t invite'
+          },
+          members: 7,
+          votes: 3,
+          userVote: true
+        },
```

Comment:
> About the actual prop format: this is a great starting point, and will probably change when I get there, but that will be another PR :) I would however unwrap it and have separate props instead of one proposal object, so that props validation can be more helpful and can work as a kind of self-documenting code, making what the component's expecting more explicit - @aniko,

What do you think about something similar to this? Should I change it now or leave it to your PR?

```
* type: String - ‘mincome' || ‘rule' || ‘member'
* ctas: Object { for: String, against: String }
* title: String
* text: String
* detailed: String
* votes: Object { total: Number, received: Number }
* hasVoted: Bool
* ownProposal: Bool
* value: Number (if type is ‘mincome’ or ‘rule’)
* picture: String (if type is ‘member’)
```

Reply:
looks great! 

the only thing I'm still thinking about is whether to break up the remaining 2 objects as well - to ctaFor and ctaAgainst, votesTotal and votes Received. this way it would be easier to prop-validate them (as vue can't yet validate types for nested props), but it also looks a bit more bloated. what do you think?

you can leave this for now, I will use this comment thread as a reference when I get here from the container side and change it in that PR!

Reply:
I think we should have a better props validator, like prop-types on react. because Objects exist and should be used when it makes sense, has it is on these cases... :/

Nevertheless, yes, we can leave it as it is to close this MR and you can finally start the container side! What do you think @taoeffect ?